src/Data/Nat/Binary/Properties.agda

```diff
@@ -12,11 +12,16 @@ open import Algebra.Bundles
 open import Algebra.Morphism.Structures
 import Algebra.Morphism.MonoidMonomorphism as MonoidMonomorphism
 open import Algebra.Consequences.Propositional
+open import Data.Bool.Base using (if_then_else_; Bool; true; false)
+open import Data.Maybe.Base using (Maybe; just; nothing)
 open import Data.Nat.Binary.Base
 open import Data.Nat as ℕ using (ℕ; z≤n; s≤s)
+import Data.Nat.DivMod as ℕ÷
```

Comment:
I'd probably just import this as `ℕ` rather than `ℕ÷`

Reply:
Explicitly imported `_/_`, `_%_`, `m/n≤m` and `+-distrib-/-∣ˡ`.